pkg/common/cns-lib/vsphere/utils.go

```diff
@@ -48,6 +48,19 @@ func IsNotFoundError(err error) bool {
 	return isNotFoundError
 }
 
+// IsAlreadyExists checks if err is the AlreadyExists fault, if yes then returns true else return false
+func IsAlreadyExists(err error) (bool, string) {
+	isAlreadyExistsError := false
+	objectName := ""
+	if soap.IsSoapFault(err) {
+		_, isAlreadyExistsError = soap.ToSoapFault(err).VimFault().(types.AlreadyExists)
+		if isAlreadyExistsError {
+			objectName = soap.ToSoapFault(err).VimFault().(types.AlreadyExists).Name
+		}
+	}
+	return isAlreadyExistsError, objectName
```

Comment:
is objectName a FCD ID?

Reply:
It is a `vStorageObject.Config.Id.Id` Same as : https://github.com/kubernetes-sigs/vsphere-csi-driver/pull/701/files#diff-183d250abe31c121fbde20d3d2b6a518444aecac24ab239e297bd632b7d86378R985